tests/benchmark/GemmBench.cpp

```diff
@@ -50,16 +51,26 @@ struct GemmParam {
 
 class GemmBench : public Benchmark {
   GemmParam param_;
-  PlaceholderBindings bindings_;
+  ExecutionContext context_;
+  PlaceholderBindings &bindings_;
   std::unique_ptr<runtime::HostManager> hostManager_;
 
 public:
-  explicit GemmBench(GemmParam param_) : param_(param_) {}
+  explicit GemmBench(GemmParam param_)
+      : param_(param_), bindings_(*context_.getPlaceholderBindings()) {}
 
   void addGemmNode(std::unique_ptr<Module> &mod, Function *fn,
                    GemmParam param) {
     auto *input = mod->createPlaceholder(param.dtype_, {param.m_, param.k_},
                                          "input", false);
+    if (param.dtype_ == ElemKind::Float16Ty) {
+      bindings_.allocate(input)->getHandle<float16>().randomize(-1.f, 1.f,
+                                                                mod->getPRNG());
+    } else {
+      assert(param.dtype_ == ElemKind::FloatTy);
+      bindings_.allocate(input)->getHandle<float>().randomize(-1.f, 1.f,
+                                                              mod->getPRNG());
+    }
```

Comment:
Huh, were these not allocated/initialized before? How did this work correctly previously? CC: @mjanderson09

Reply:
No, they were not initialized before.  The ExecutionContext was also being passed to runInference in such a way that it wouldn't matter if they were initialized since these bindings weren't being passed in anyway.  That's what motivated my changes to run().

Reply:
And also, the outputs are not checked, so it doesn't matter what the inputs are :)